code/src/utils/set-product-size/setProductsPerPageSize.ts

```diff
@@ -0,0 +1,13 @@
+const setProductsPerPageSize = (screenWidth: number) => {
+  if (screenWidth >= 1536) {
+    return 10;
+  } else if (screenWidth >= 1200) {
+    return 12;
+  } else if (screenWidth >= 900) {
+    return 9;
+  } else {
+    return 10;
+  }
+};
+
+export default setProductsPerPageSize;
```

Comment:
try to use `switch case` statement here
It'll be more readable and also good practice with that

Reply:
Yeah, and maybe let's create the constant with these breakpoints and move it to the `constants` folder. Name it breakpoints.ts

Reply:
I moved the Breakpoints to constants folder and refactored the function to a switch statement.